samples/opengl/cube/cube.cpp

```diff
@@ -458,9 +458,10 @@ wxEND_EVENT_TABLE()
 MyFrame::MyFrame( bool stereoWindow )
        : wxFrame(NULL, wxID_ANY, "wxWidgets OpenGL Cube Sample")
 {
-    int stereoAttribList[] = { WX_GL_RGBA, WX_GL_DOUBLEBUFFER, WX_GL_STEREO, 0 };
+    int attribList[] = { WX_GL_RGBA, WX_GL_DOUBLEBUFFER, WX_GL_DEPTH_SIZE, 16 };
```

Comment:
This list should be [0-terminated](https://docs.wxwidgets.org/latest/classwx_g_l_canvas.html#a56d0a2a022c0c34b7efa1114da151177), on Windows the cube doesn't even render because of this.

Maybe this is a good moment to start using `wxGLAttributes` here too instead of an `attribList`.

Reply:
Fixed.